### DISCUSSION TITLE
A command like `listforwards`

### DISCUSSION THREAD

**Discussion by fiatjaf** [ACINQ/eclair#2043] [Ideas]
Does Eclair keep track of forward attempts and successes?

If yes, would be great to have a command to list those.

If not, maybe just a command to display how many channel updates a channel had. I guess this already exists.

**Comment by fiatjaf** [comment]
This command shows the number of HTLCs that have been attempted I suppose: `eclair channels | jq '.[].data.commitments | {remoteNextHtlcId, localNextHtlcId}'`

But there is probably more that can be inferred from the `channels` data that I'm missing, right?

**Reply by t-bast** [reply]
Have you tried the `audit` API (https://acinq.github.io/eclair/#audit) and the `channelstats` API (https://acinq.github.io/eclair/#channelstats)?
You can also directly read from the `audit` database, but it may be a bit more work to decode the content (IIRC with postgres you have everything in json format, but not with sqlite).
